src/routes/WelcomeWizardRouter/AddLocation/AddLocation.js

```diff
@@ -46,7 +47,7 @@ export default class AddLocation extends Component {
 
     this.props.updateUserSettings(changes)
       .then(() => {
-        this.props.trackAnalyticsEvent('Signup Complete')
```

Comment:
Without a careful analysis, I wonder if "Signup Profile Finished" (the value of `SIGNUP_PROFILE_FINISHED`) vs the previous  "Signup Complete" will create some discontinuity in analytics data previous to this change. I do see that we still send "Signup Complete" later now.

Reply:
I mean it's slightly different, now we call Signup Complete when they have registered, instead of waiting until they have filled in Profile photo and location (when we call SIGNUP_PROFILE__FINISHED. So yes the same event will mean something slightly different now, but I think thats ok. Moving forward this is more accurate.

Reply:
we actually send Signup Complete sooner now

Reply:
I know this is a bit confusing, and hard to track but within the Web and the similarly in the App the `AuthState` object reflects as `AuthState.SignupInProgress` (as `user.signupInProgress` is still true) where you're currently sending  `AnalyticsEvents.SIGNUP_COMPLETE`. This `signupInProgress` indicator means the user will be  redirected back into the signup process upon login if they leave and return. 

* I think we should be sending `AnalyticsEvents.SIGNUP_COMPLETE` only once `signupInProgress` is set to false to keep things consistent between these two worlds. 

* Where you're currently sending `AnalyticsEvents.SIGNUP_COMPLETE` I would create an event called `AnalyticsEvents.REGISTRATION_COMPLETE`. 

* Consider removing `SIGNUP_` from `SIGNUP_EMAIL_VERIFICATION_SENT` and `SIGNUP_EMAIL_VERIFIED`?

Reply:
Sorry to complicate this at all, but if you follow and agree, I think this is one of those spots where we can avoid creating accidental complexity / unnecessary layers or indirection where we may thank ourselves later.

Reply:
I'm fine with all that

Reply:
though I do kind of like having SIGNUP_ in front of all 4 of those events because they are all part of the signup funnel, not attached though

Reply:
so then SIGNUP_REGISTRATION_COMPLETE and SIGNUP_COMPLETE

Reply:
Cool with that.

Reply:
though, the problem with this is it is harder for someone to look at and understand, we'll have to explain to people. this is why I switched it to SIGNUP_PROFILE_FINISHED 🤷‍♂️

Reply:
though, the problem with this is it is harder for someone to look at and understand, we'll have to explain to people. this is why I switched it to SIGNUP_PROFILE_FINISHED 🤷‍♂️

Reply:
Yeah, SIGNUP_COMPLETE means exactly that in the ÅuthState world, and I would think we should keep that event name for the final step.